chaincode/traintuple.go

```diff
@@ -203,6 +203,9 @@ func (traintuple *Traintuple) Save(db LedgerDB, traintupleKey string) error {
 		if err := db.CreateIndex("traintuple~computeplanid~worker~rank~key", []string{"traintuple", traintuple.ComputePlanID, traintuple.Dataset.Worker, strconv.Itoa(traintuple.Rank), traintupleKey}); err != nil {
 			return err
 		}
+		if err := db.CreateIndex("traintuple~computeplanid", []string{"computeplan", traintuple.ComputePlanID}); err != nil {
```

Comment:
See the above comment on this index.

Reply:
Fixed https://github.com/SubstraFoundation/substra-chaincode/pull/22/commits/625d2009351395cedb567a453bee1ec30a636ae4